packages/firebase_ml_vision/example/lib/camera_preview_scanner.dart

```diff
@@ -31,6 +31,8 @@ class _CameraPreviewScannerState extends State<CameraPreviewScanner> {
   final TextRecognizer _recognizer = FirebaseVision.instance.textRecognizer();
   final TextRecognizer _cloudRecognizer =
       FirebaseVision.instance.cloudTextRecognizer();
+  final DocumentTextRecognizer _cloudDocumentRecognizer =
```

Comment:
nit: maybe `_cloudDocumentTextRecognizer`?

Reply:
This name was chosen to be consistent with that of TextRecognizer:
TextRecognizer has _cloudRecognizer, and therefore I named DocumentTextRecognizer _cloudDocumentRecognizer.

I would like to have consistent naming in the app. However, if you disagree I will change it.